RecoTauTag/RecoTau/test/rerunMVAIsolationOnMiniAOD_Phase2.py

```diff
@@ -2,17 +2,17 @@
 
 process = cms.Process("rerunMVAIsolationOnMiniAODPhase2")
 
-process.load('Configuration.Geometry.GeometryExtended2026D41Reco_cff')
+process.load('Configuration.Geometry.GeometryExtended2026D49Reco_cff')
 process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
 from Configuration.AlCa.GlobalTag import GlobalTag
 process.GlobalTag = GlobalTag(process.GlobalTag, '106X_upgrade2023_realistic_v2')
```

Comment:
this Global Tag is awfully outdated for this geometry, perhaps it doesn't matter for taus, but `auto:phase2_realistic_T15` is for sure a better maintained choice.

Reply:
Thank you for this comment. A GlobalTag choice has indeed a marginal, if any, effect for this tauID, but it is a good idea to use something better maintainable as auto conditions. The PR will be updated following your suggestion.